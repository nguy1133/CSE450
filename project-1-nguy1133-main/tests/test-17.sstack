// Variables should be printable alongside other values.
PUSH "42"
POP value
PRINT "The variable 'value' is set to be " value
