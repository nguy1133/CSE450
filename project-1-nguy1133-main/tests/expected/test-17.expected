The variable 'value' is set to be 42
